### PR TITLE
Route view

### PR DESCRIPTION
**Pull request by Cindyzhang977**
Compile components into route page view. Adjusted some styles on individual components. 
(Note: will modularize css in later PR)

image

### CODE REVIEW

File: frontend/src/RouteView.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
+import React from 'react';
```
Comment (mecolmg) [inline frontend/src/RouteView.js:1]: Have you all agreed that the root views will be stored in the src directory, or should it be put in a subdirectory (i.e. routeview or just route)?

Comment (zoebarth) [reply frontend/src/RouteView.js:1]: I think we decided to store them directly in src

File: frontend/src/App.js [hunk 2]
```diff
@@ -16,6 +18,7 @@ function App() {
         Click Me!
       </Button>
       <div>{text}</div>
+      <RouteView />
```
Comment (zoebarth) [inline frontend/src/App.js:21]: Are we planning on leaving this here?

Comment (Cindyzhang977) [reply frontend/src/App.js:21]: oh yeah we should prob remove the button

**Pull request merged**